CODEOWNERS

```diff
@@ -206,13 +206,15 @@
 /include/sys_io.h                         @andrewboie @andyross
 /include/toolchain.h                      @andrewboie @andyross @nashif
 /include/toolchain/                       @andrewboie @andyross
+/include/updatehub.h                      @chtavares592 @otavio
 /include/zephyr.h                         @andrewboie @andyross
 /kernel/                                  @andrewboie @andyross
 /lib/gui/                                 @vanwinkeljan
 /lib/posix/                               @pfalcon
 /kernel/device.c                          @andrewboie @andyross @nashif
 /kernel/idle.c                            @andrewboie @andyross @nashif
 /samples/basic/servo_motor/*microbit*     @jhe
+/lib/updatehub/                           @chtavares592 @otavio
```

Comment:
What about placing the updatehub into `subsys/net/lib` directory?

Reply:
As discussed in previous comments, we are open to moving it to somewhere else when more OTA stuff gets in, but it is not limited to net use, so we prefer to not move it now.

Reply:
> but it is not limited to net use, so we prefer to not move it now.

Sure, np